File: Solutions/1-two-sum/two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int[] result = new int[2];  // Array of size 2 to store indices
        
        // Brute force checking to find the solution
        for (int i = 0; i < nums.length; i++) {
            for (int j = i + 1; j < nums.length; j++) {
                if (nums[i] + nums[j] == target) {
                    result[0] = i;
                    result[1] = j;
                    return result;
                }
            }
        }

        return null;  // If no solution is found, return null
    }
}

File: Solutions/1-two-sum/two-sum.kt
class Solution {
    fun twoSum(nums: IntArray, target: Int): IntArray? {
        val result = IntArray(2)  // Array of size 2 to store indices
        
        // Brute force checking to find the solution
        for (i in nums.indices) {
            for (j in i + 1 until nums.size) {
                if (nums[i] + nums[j] == target) {
                    result[0] = i
                    result[1] = j
                    return result
                }
            }
        }

        return null  // If no solution is found, return null
    }
}